lib/capistrano/configuration.rb

```diff
@@ -27,6 +27,10 @@ def set(key, value)
       config[key] = value
     end
 
+    def append(key, value)
+      set(key, fetch(key, []) << value)
```

Comment:
Should we check that `fetch(key)` actually returns an array before using the `<<` operator?
Something other than an array might behave in different/unexpected ways...